cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -227,11 +199,11 @@ int main(int argc, char** argv)
 
     if (long_read)
     {
-        generate_simulated_long_reads(windows, batch_size);
+        generate_window_data("/sample-bonito.txt", 8, 6, windows, batch_size);
```

Comment:
let's create the filename outside of the function, and pass in the full path to the file into the function

i.e. move the `const std::string input_data = std::string(CUDAPOA_BENCHMARK_DATA_DIR) + file_name;` from the function definition into this conditional

Reply:
done.